src/bin.js

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env node
-const { exit } = require('process')
+import { exit, argv } from 'process'
```

Comment:
can we use the `node:` scoped imports? As it is recommended by node to use them. – as it marks it as built-in module and puts an additional safety layer on top of it.

I personally like the fact to differentiate really fast between builtins and other packages.

Reply:
Yes, great idea!

We should also use [`unicorn/prefer-node-protocol`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-node-protocol.md) to enforce it and also auto-fix it.

In the past, both `eslint-plugin-import` and `eslint-plugin-node` used to have issues with the `node:` scheme. Just in case this is still the case, and to keep PRs small, we might want to do it as a second step once all modules have been migrated to pure ES modules. What do you think?

Reply:
Opened an issue at https://github.com/netlify/team-dev/issues/39 and added it to the todo list of pure ES modules support (https://github.com/netlify/team-dev/issues/36).